packages/e2e-tests/scripts/buildkite_run_fe_tests.ts

```diff
@@ -255,6 +255,7 @@ export default async function run_fe_tests(
   }
   // process.env.RECORD_REPLAY_DIRECTORY =
 
+  process.env.REPLAY_ENABLE_ASSERTS = process.env.RECORD_REPLAY_ENABLE_ASSERTS = "1";
```

Comment:
Do we really need to specify both `REPLAY_ENABLE_ASSERTS` and `RECORD_REPLAY_ENABLE_ASSERTS`? (Is there anything that doesn't support `REPLAY_ENABLE_ASSERTS`?)

Reply:
We have not done the rename yet. Its right here: https://linear.app/replay/issue/TT-1281/rename-record-replay-enable-asserts-env-variable